flink-core/src/main/java/org/apache/flink/util/FileUtils.java

```diff
@@ -116,6 +118,14 @@ else if (file.exists()) {
 			}
 			catch (NoSuchFileException e) {
 				// if the file is already gone (concurrently), we don't mind
+			} catch (AccessDeniedException e) {
```

Comment:
Can we special case this also for Windows? Should never occur on Unix (and should throw an exception if it happens)...